va/va.h

```diff
@@ -238,6 +238,17 @@ typedef int VAStatus;	/** Return status type from functions */
 #define VA_PADDING_LARGE        32
 
 /**
+ * This is to indicate that the color-space conversion uses full range or reduced range.
+ * VA_SOURCE_RANGE_FULL(Full range): Y/Cb/Cr is in [0, 255]. It is mainly used
+ *      for JPEG/JFIF formats. The combination with the BT601 flag means that
+ *      JPEG/JFIF color-space conversion matrix is used.
+ * VA_SOURCE_RANGE_REDUCED(Reduced range): Y is in [16, 235] and Cb/Cr is in [16, 240].
+ *      It is mainly used for the YUV->RGB color-space conversion in SDTV/HDTV/UHDTV.
```

Comment:
Can this be used to get limited range RGB as well?  (As used in HDMI.)

Maybe mention the ranges for 10-bit YUV in the comment, too.

Reply:
I suppose we can add more color range, but I wonder whether we should add them now? who are using it?

Reply:
I think Full range and Reduced range can be used for both RGB and YUV and also 8-bit/10-bit.